lib/rules/validate-jsdoc/require-param-description.js

```diff
@@ -0,0 +1,31 @@
+module.exports = validateParamTagDescription;
+module.exports.tags = ['param', 'arg', 'argument'];
+module.exports.scopes = ['function'];
+module.exports.options = {
+    requireParamDescription: {allowedValues: [true]}
+};
+
+/**
+ * Validator for @param
+ *
+ * @param {(FunctionDeclaration|FunctionExpression)} node
+ * @param {DocTag} tag
+ * @param {Function} err
+ */
+function validateParamTagDescription(node, tag, err) {
+    // checking existance
+    if (!tag.description) {
```

Comment:
Good point!   Do you want me to change the require parameter type.  (That
is what I copied this from)
On Jun 15, 2015 12:42 PM, "Alexej Yaroshevich" <notifications@github.com>
wrote:

> In lib/rules/validate-jsdoc/require-param-description.js
> <https://github.com/jscs-dev/jscs-jsdoc/pull/111#discussion_r32452339>:
>
> > +module.exports.tags = ['param', 'arg', 'argument'];
> > +module.exports.scopes = ['function'];
> > +module.exports.options = {
> > +    requireParamDescription: {allowedValues: [true]}
> > +};
> > +
> > +/**
> > + * Validator for @param
> > + *
> > + * @param {(FunctionDeclaration|FunctionExpression)} node
> > + * @param {DocTag} tag
> > + * @param {Function} err
> > + */
> > +function validateParamTagDescription(node, tag, err) {
> > +    // checking existance
> > +    if (!tag.description) {
>
> I'd do here early return because we don't need to check "nothing" ;-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jscs-dev/jscs-jsdoc/pull/111/files#r32452339>.
>